contracts/common/MiniMeToken.sol

```diff
@@ -131,7 +131,7 @@ contract MiniMeToken is Controlled {
         uint8 _decimalUnits,
         string _tokenSymbol,
         bool _transfersEnabled
-    ) {
+    )  public {
```

Comment:
Linter is complaining here in the CI:  ERROR: Function 'MiniMeToken': Opening brace must be on a new line immediately after the last modifier.

Reply:
Fixed that error all CI tests are now passing.